tests/testthat/test_resequence.R

```diff
@@ -1,9 +1,23 @@
-data <- data.frame(Sign = c("2000-01-01", "2001-01-01", "2001-01-01_2000-01-01", "2000-01-01", NA, "-01-01"),
-                   Force = c("2001-01-01", "2000-01-01", "2001-01-01", NA, "2001-01-01", "9999"))
-result <- as.matrix(unname(data.frame(c("2000-01-01", "2000-01-01", "2000-01-01", "2000-01-01", NA, "-01-01"),
-                                      c("2001-01-01", "2001-01-01", "2001-01-01", NA, "2001-01-01", "9999"))))
+data <- data.frame(Sign = c("2000-01-01", "2001-01-01",
+                            "2001-01-01_2000-01-01",
+                            "2000-01-01", NA, "-01-01"),
+                   Force = c("2001-01-01", "2000-01-01",
+                             "2001-01-01", NA,
+                             "2001-01-01", "9999"))
```

Comment:
Are we still using resequence anywhere? How does it handle messydates? Maybe move this to messydates and make it more robust?